data/json/itemgroups/supplies.json

```diff
@@ -22,6 +22,86 @@
       [ "motor_micro", 40 ]
     ]
   },
+  {
+    "type": "item_group",
+    "id": "hardware_bulk",
+    "//": "large quantities of common building supplies.",
+    "items": [
+      { "item": "2x4", "prob": 100, "count": [ 5, 20 ] },
+      { "item": "rock", "prob": 70, "count": [ 10, 20 ] },
+      { "item": "coal_lump", "prob": 20, "count": [ 1, 10 ] },
+      [ "material_shrd_limestone", 70 ],
+      [ "fertilizer_commercial", 100 ],
+      [ "material_quicklime", 70 ],
+      [ "material_sand", 100 ],
+      [ "concrete", 70 ],
+      { "item": "rebar", "prob": 40, "count": [ 5, 10 ] },
+      { "item": "brick", "prob": 40, "count": [ 5, 10 ] },
+      { "item": "pipe", "prob": 40, "count": [ 5, 10 ] },
+      { "item": "nail", "prob": 60, "count": [ 500, 1000 ] }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "lumberyard",
+    "subtype": "collection",
+    "items": [
+      {
+        "distribution": [
+          { "item": "2x4", "prob": 500, "count": [ 20, 50 ] },
+          { "item": "log", "prob": 200, "count": [ 5, 10 ] },
+          { "item": "stick_long", "prob": 100, "count": [ 10, 20 ] },
+          { "item": "stick", "prob": 50, "count": [ 5, 10 ] },
+          { "item": "nail", "prob": 100, "count": [ 500, 1000 ] }
+        ],
+        "prob": 100
+      },
+      [ "pine_bough", 10 ]
+    ]
+  },
+  {
+    "id": "supplies_hardware",
+    "type": "item_group",
+    "//": "General purpose supplies for building / blue collar work.",
+    "//2": "Stuff you might find in a jobsite, a hardware store, or a home workshop.",
+    "items": [
+      [ "concrete", 20 ],
+      { "item": "rebar", "prob": 20, "count": [ 5, 10 ] },
+      { "item": "brick", "prob": 20, "count": [ 5, 10 ] },
+      [ "superglue", 30 ],
+      [ "chain", 20 ],
+      [ "rope_6", 4 ],
+      [ "rope_30", 35 ],
+      [ "circsaw_blade", 8 ],
+      [ "glass_sheet", 5 ],
+      { "item": "pipe", "prob": 20, "count": [ 5, 10 ] },
+      { "item": "nail", "prob": 60, "count": [ 200, 400 ] },
```

Comment:
A single "count" of nails is 100 nails. A count of 200 here will spawn 20,000 (!) nails.